tests/integration_tests/bugs/test_lp1835584.py

```diff
@@ -0,0 +1,88 @@
+""" Integration test for LP #1835584
+
+Kernels prior to 4.15 report dmi product_uuid as uppercase, Azure
+relies on system UUID as their "instance-id". Upgrades to kernels >= 4.15
+will represent the same product_uuid as lowercase. System UUID should be a
+case-insensitive comparison and avoid triggering "new instance" detection
+in cloud-init on Azure platform.
+
+The test will launch an Ubuntu image which has a 5.4 kernel. We allow
```

Comment:
This isn't true: this test is not constrained by Ubuntu release, so will launch whatever version of the kernel is in the latest Azure image for the target Ubuntu release.

Reply:
We've now pinned the test to launch a specific Ubuntu PRO FIPS image that will be available forever and exhibits a UUID case change when upgrading to linux-azure from linux-azure-fips.